### PR TITLE
docs: add Annie Wang to contributors.json

### PR DESCRIPTION
**Pull request by annieyw**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]
Lgtm! Welcome!

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -835,5 +835,12 @@
     "bio": "Manu heads technical program management for Angular at Google. Manu keeps the big picture in focus and works with cross-functional teams to plan, execute and usher programs through the entire lifecycle.",
     "groups": ["Angular"],
     "lead": "juleskremer"
+  },
+  "annieyw": {
```
Comment (IgorMinar) [inline aio/content/marketing/contributors.json:839]: Oops.. hold on. This is your Google ldap and not your GitHub username. Can you please change this and the image name to "vxlydia"? Thanks

**Review by IgorMinar — CHANGES_REQUESTED** [review]

**Review by IgorMinar — COMMENTED** [review]
That looks better. Thanks Annie.

Can you please do my one more favor and use got to squash all the 3 commits into a single commit that follows our commit message conventions: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit ?

Thanks

**Review by gkalpak — COMMENTED** [review]
Drive-by comment: We also try to keep images less than 30KB in size 🙏

**Comment by vxlydia** [comment]
Squashed commits and made image smaller.

**Review by IgorMinar — APPROVED** [review]
lgtm now! thank you for the updates!

**Comment by IgorMinar** [comment]
weirdly I see that the CI failed with an error that is not related to your change. It might be just a transient error (which is not great, but occasionally happens). I restarted the failed CI builds, so hopefully the errors will be resolved. If the CI fails again, can you please rebase your PR and force push into this PR branch, if that doesn't help (I really hope this doesn't happen, but occasionally it does), please post on the #fw-caretaker channel and ask for help there. Thanks!

**Comment by IgorMinar** [comment]
@vxlydia oh.. great! CI is green! so you are good to go, the caretaker will pick up this PR today/tomorrow and merge it.

If you do need to rebase and force push any of your PRs in the future, you can find the docs on how to do so here: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-a-pull-request-pr

**Comment by IgorMinar** [comment]
uh.. the CLA bot freaked out.

**Comment by IgorMinar** [comment]
@annieyw Unfortunately this PR now needs a rebase, that's why the CI is failing (the reason is in one of the CI logs linked to from the ["Details" link above](https://circleci.com/gh/angular/angular/687204?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). Can you please follow the instructions I posted earlier:

> If you do need to rebase and force push any of your PRs in the future, you can find the docs on how to do so here: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-a-pull-request-pr

Addinally because your PR was created with your previous github username, but during the course of the life of this PR the username has changed, the CLA bot is freaking out. This affects only this one PR (or any other PRs that were outstanding while you changed your username). You can safely ignore this warning, and manually remove the "CLA: no" label and add "CLA: yes" label (as the bot instructs you to do in the long comment above). You will have to do this every time you push into this branch for this PR (hopefully just once when you are rebasing).

I'm sorry about all this trouble.. it's unfortunate that all these things are happening on your first PR. I hope that your contributing experience to our repos will only get better from here on.

**Comment by annieyw** [comment]
@IgorMinar Seems like all the checks are good now. Thank you for guiding me through this process and sorry for all the trouble!

**Comment by IgorMinar** [comment]
@annieyw yay! Finally! No need to apologize!

This PR was hit by a perfect storm of almost all the possible obstacles that cause angular/angular PRs to get stuck and pile up.

I'm glad that we were able to work through this and get it green. Now that the PR is green, the @angular/angular-caretaker (an Angular team member on "caretaker" rotation) will pick it up and merge it.

Hopefully this will happen early tomorrow during PT office hours unless we have some google3 or CI outages going on. Let's hope that will not be the case... Almost there...🤞

**Pull request closed without merging**